consensus/polybft/signer/common.go

```diff
@@ -0,0 +1,139 @@
+package bls
+
+import (
+	"crypto/rand"
+	"crypto/sha256"
+	"errors"
+	"fmt"
+	"math/big"
+
+	bn256 "github.com/umbracle/go-eth-bn256"
+)
+
+var (
+	zeroBigInt = *big.NewInt(0)
+	oneBigInt  = *big.NewInt(1)
+	g2         = *new(bn256.G2).ScalarBaseMult(&oneBigInt)  // Generator point of G2 group
+	zeroG1     = *new(bn256.G1).ScalarBaseMult(&zeroBigInt) // Zero point in G2 group
+	zeroG2     = *new(bn256.G2).ScalarBaseMult(&zeroBigInt) // Zero point in G2 group
+
+	// this is a copy of point p is a prime over which we form a basic field: 36u⁴+36u³+24u²+6u+1.
+	// TODO delete this point and use bn256.p
+	bn256P, _ = new(big.Int).SetString("21888242871839275222246405745257275088696311157297823662689037894645226208583", 10)
+)
+
+// genRandomBytes generates byte array with random data
+func genRandomBytes(size int) (blk []byte) {
+	blk = make([]byte, size)
+
+	_, err := rand.Reader.Read(blk)
+	if err != nil {
+		panic("Cannot generate message for tests. This should never happen")
+	}
```

Comment:
Please don't introduce panics into the codebase, you can handle this as an error - it's not a runtime level exception

Reply:
This is a method that should never panic and that is used for tests only. What are your concerns exactly? Is it with the panic itself on how it is being used in this context?